database/bfgd/postgres/postgres.go

```diff
@@ -965,6 +965,64 @@ func (p *pgdb) AccessPublicKeyDelete(ctx context.Context, publicKey *bfgd.Access
 	return nil
 }
 
+// BtcBlocksHeightsWithNoChildren returns the heights of blocks that have no
+// child blocks in our database, these represent possible forks that we have
+// not handled yet
+func (p *pgdb) BtcBlocksHeightsWithNoChildren(ctx context.Context) ([]uint64, error) {
+	log.Tracef("BtcBlocksHeightsWithNoChildren")
+	defer log.Tracef("BtcBlocksHeightsWithNoChildren exit")
+
+	/*
+		give me all heights for btc blocks
+		where there does not exist a child of this block
+		and there are no other blocks at this height with children
+		exclude the tip because it won't have children by definition
+	*/
+	const q = `
+	SELECT height FROM btc_blocks bb1
+
+	WHERE NOT EXISTS (SELECT * FROM btc_blocks bb2 WHERE SUBSTR(bb2.header, 5, 32) = bb1.hash)
+	
+	AND NOT EXISTS (
+		SELECT * FROM btc_blocks bb3 WHERE bb1.height = bb3.height 
+		AND EXISTS(
+			SELECT * FROM btc_blocks bb4 WHERE substr(bb4.header, 5, 32) = bb3.hash
+		)
+	)
+	ORDER BY HEIGHT DESC
+	OFFSET $1 + 1
+	LIMIT 100
+	`
+
+	heights := []uint64{}
```

Comment:
nit: use nil slice
```suggestion
	var heights []uint64
```

Reply:
hey @joshuasing I did this on purpose; if there are no results, I just wanted to return an empty slice, as opposed to `nil` (so you could check `if len(result/heights) == 0` for example), do you feel strongly that this should be `nil`?